BigDatasets_details.md

```diff
@@ -0,0 +1,5 @@
+## Big Datasets
```

Comment:
What is this file for? There's already a BigDatasets.md file

Reply:
To expand on what the training will cover. I don't know enough to expand on details myself.